.devcontainer/Dockerfile

```diff
@@ -4,6 +4,9 @@ FROM mcr.microsoft.com/dotnet/sdk:5.0
 RUN apt-get update
 RUN apt-get install -y gnupg
 
+# VIM
+RUN apt-get install -y vim
```

Comment:
Codacy found an issue: [Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`](https://app.codacy.com/gh/LukasReschke/AuthServer/pullRequest?prid=6395991)